docs/website/src/pages/index.tsx

```diff
@@ -0,0 +1,135 @@
+            <div className={styles.indexCtas}>
+              <Link className="button button--primary" to="/docs/intro">
+    Read the docs
+              </Link>
+              <Link className="button button--info" to="/docs/getting-started/quickstart">
+    Quick start
+              </Link>
+              <span className={styles.indexCtasGitHubButtonWrapper}>
+                <iframe
+                  className={styles.indexCtasGitHubButton}
+                  src="https://ghbtns.com/github-btn.html?user=redhat-developer&amp;repo=odo&amp;type=star&amp;count=true&amp;size=large"
+                  width={160}
+                  height={30}
+                  title="GitHub Stars"
+                  frameBorder={0}
+                />
+              </span>
+            </div>
+          </div>
+        </div>
+        <div className={clsx(styles.title, styles.titleDark)}>
+          <div className={styles.titleInner}>
+            Visit our <Link to="/blog"> our blog </Link>for the newest developments on odo
+
+          </div>
+        </div>
+        <div className={styles.overview}>
+          <div className="container text--center margin-top--md">
+            <div className="row">
+              <div className="col col--5 col--offset-1">
+                    <video className={styles.loopVideo} autoPlay loop muted><source src="/video/container_ship.webm" type="video/webm"/></video>
+                <h2 className={clsx(styles.featureHeading)}>
+                  Use on <b className={styles.kubernetesFont}>Kubernetes</b> or <b className={styles.openshiftFont}>OpenShift</b>
+                </h2>
+                <p className="padding-horiz--md">
+    We provide first class support for both Kubernetes and OpenShift. Choose your favourite orchestrator and deploy your application.
+                </p>
+              </div>
+              <div className="col col--5">
+                    <video className={styles.loopVideo} autoPlay loop muted><source src="/video/coding.webm" type="video/webm"/></video>
+                <h2 className={clsx(styles.featureHeading)}>
+                  Deploy fast and often
+                </h2>
+                <p className="padding-horiz--md">
+    Spend less time maintaining your deployment infrastructure and more time coding. Immediately have your application running each time you compile.
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div className={clsx(styles.overview, styles.overviewAlt)}>
+          <div className="container text--center margin-top--lg">
+            <div className="row">
+              <div className="col">
+                <h2 className={clsx(styles.featureHeading)}>
+    Standalone client
+                </h2>
+                <p className="padding-horiz--md">
+    <code>odo</code> is a standalone tool that communicates directly with the container orchestrator API. There is no requirement for a daemon or server process.
```

Comment:
Why not just use Kubernetes instead? I'm suggesting this because, in my head, the more times "Kubernetes" appears on the website, the more we will be able to convey that odo is a Kubernetes tool!
```suggestion
    <code>odo</code> is a standalone tool that communicates directly with the Kubernetes API. There is no requirement for a daemon or server process.
```